### DISCUSSION TITLE
Memories, misty water colored memories

### DISCUSSION THREAD

**Discussion by justageek** [zed-industries/zed#18709] [Feature Requests]
Dear Zed, please let me know when you fix your memory leaks so I can use your product.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
